.github/workflows/flutter-build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/setup-java@v1
         with:
           java-version: '12.x'
-      - run: git clone https://github.com/flutter/flutter.git --depth 1 -b v1.12.13+hotfix.5 _flutter
+      - run: git clone https://github.com/flutter/flutter.git --depth 1 -b v1.12.13+hotfix.6 _flutter
       - run: echo "::add-path::$GITHUB_WORKSPACE/_flutter/bin"
```

Comment:
Extract the flutter version to a common variable, such as:
https://github.com/argoproj/argo/blob/master/examples/arguments-parameters.yaml#L13

Also add comment:
```
Update flutter-test.yml file to use same Flutter version
```

Reply:
What I'll do is put the comment in pubspec.yaml instead.  (And not doing the variable due to your later comment)